config/samples/disk_formats/create-image.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+set -evx
+# Create a image with supported disk format
+#
+# The scripts assumes:
+#
+# 1. an available openstack cli
+# 2. glanceAPI configured with disk formats
+# 3. a single layout (file / NFS) backend and disk format is deployed
+#
+#
+TIME=5
+IMAGE_NAME="myimage-disk_format-test"
+EXIT_CODE=$?
+
+
+function create_image() {
+    # This method is create, list and delete image
+    # $1 - disk format
+
+    echo This is a dodgy image > "${IMAGE_NAME}"
+
+    # Stage 0 - Delete any pre-existing image
+    openstack image list -c ID -f value | xargs -n 1 openstack image delete
+    sleep "${TIME}"
+
+    # Stage 1 - Create image
+    openstack image create \
+        --disk-format "$1" \
+        --container-format bare \
+        "${IMAGE_NAME}"
+
+    ID=$(openstack image list | awk -v img=$IMAGE_NAME '$0 ~ img {print $2}')
+    echo "Image ID: $ID"
+    sleep "${TIME}"
+
+    if [ -z "$ID" ]
+    then
+      openstack image list -c ID -f value | xargs -n 1 openstack image delete
+      exit 1
+    fi
+
+    # Stage 2 - Check the image is active
+    openstack image list
```

Comment:
do we need `openstack image list` here? If it's just for debug purposes, I'm wondering if we want to have a `DEBUG` parameter that can be set to either 0 or 1 to trigger such commands that increase verbosity for troubleshooting purposes.

Reply:
I see you resolved this. Thank you.